### ISSUE TITLE
gaps in old files

### ISSUE THREAD

**Issue by emylonas** [Brown-University-Library/iip-texts#17]
We have 64 cases (in about 50 files) where the original conversion to Epidoc left the following information: 
```
1. <gap reason="illegible"><?human @extent=4-6 @unit=characters ?></gap>
2. <gap reason="illegible"><?human @extent=ca. 6 @unit=characters ?></gap>
3.  <gap reason="illegible"><?human @extent=ca. 17-18 @unit=characters ?></gap>
```

I take this to mean that at some point, an encoder or the conversion process wanted a human to look at and decide about this.

When I look at the Epidoc Guidelines for illegible text that knows roughly how many characters are missing, here: http://www.stoa.org/epidoc/gl/latest/trans-illegiblecharapprox.html

it suggests 
`<gap reason="illegible" atLeast="4" atMost="6" unit="character"/>`
or
`<gap reason="illegible" quantity="7" unit="character" precision="low"/>`

I think we can easily convert case 1) to the "atLeast-atMost" format and case 2) to the "precision=low" format. However, the Guidelines don't seem to allow or encourage the use of `@precision` with the `@atLeast` and `@atMost` method. I can query the list, or we can just decide to go one way or another.

I can fix these globally, but checking on how to go about it. List of files below:
```
caes0008.xml  <gap reason="illegible"><?human @extent=14-16 @unit=characters ?></gap>
caes0008.xml  <gap reason="illegible"><?human @extent=10-12chars ?></gap>
caes0008.xml  <gap reason="illegible"><?human @extent=4-6 @unit=characters ?></gap>
caes0008.xml  <gap reason="illegible"><?human @extent=8-10 @unit=characters ?></gap> 
caes0008.xml  <gap reason="illegible"><?human @extent=7-9 @unit=characters ?></gap>
caes0008.xml  <gap reason="illegible"><?human @extent=10-12 @unit=characters ?></gap>
caes0013.xml  <gap reason="illegible"><?human @extent=9-11chars ?></gap>
caes0013.xml  <gap reason="illegible"><?human @extent=4-6 @unit=characters ?></gap>
caes0013.xml  <gap reason="illegible"><?human @extent=2-4 @unit=characters ?></gap>
caes0013.xml  <gap reason="illegible"><?human @extent=3-5 @unit=characters ?></gap>
caes0020.xml  <gap reason="illegible"><?human @extent=1-3 @unit=characters ?></gap>
caes0020.xml  <gap reason="illegible"><?human @extent=1-3 @unit=characters ?></gap>
caes0023.xml  <gap reason="illegible"><?human @extent=3-5 @unit=characters ?></gap>
caes0026.xml  <gap reason="illegible"><?human @extent=6-8 @unit=characters ?></gap>
caes0026.xml  <gap reason="illegible"><?human @extent=8-10 @unit=characters ?></gap>
caes0026.xml  <gap reason="illegible"><?human @extent=7-9 @unit=characters ?></gap>
caes0026.xml  <gap reason="illegible"><?human @extent=6-8 @unit=characters ?></gap>
caes0026.xml  <gap reason="illegible"><?human @extent=7-9 @unit=characters ?></gap>
caes0026.xml  <gap reason="illegible"><?human @extent=9-11 @unit=characters ?></gap>
caes0062.xml  <gap reason="illegible"><?human @extent=4-6 @unit=characters ?></gap>
caes0063.xml  <gap reason="illegible"><?human @extent=11-13 @unit=characters ?></gap>
caes0119.xml  <gap reason="illegible"><?human @extent=22-24 @unit=characters ?></gap>
caes0156.xml  <gap reason="illegible"><?human @extent=8-10 @unit=characters ?></gap>
caes0156.xml  <gap reason="illegible"><?human @extent=2-3 @unit=characters ?></gap>
caes0156.xml  <gap reason="illegible"><?human @extent=1-2 @unit=characters ?></gap>
caes0156.xml  <gap reason="illegible"><?human @extent=2-4 @unit=characters ?></gap>
caes0200.xml  <gap reason="illegible"><?human @extent=1-2 @unit=characters ?></gap>
caes0361.xml  <gap reason="illegible"><?human @extent=4-6 @unit=characters ?></gap>
caes0361.xml  <gap reason="illegible"><?human @extent=4-6 @unit=characters ?></gap>
caes0362.xml  <gap reason="illegible"><?human @extent=2-4 @unit=characters ?></gap>
caes0362.xml  <gap reason="illegible"><?human @extent=2-4 @unit=characters ?></gap>
caes0362.xml  <gap reason="illegible"><?human @extent=1-3 @unit=characters ?></gap>
caes0362.xml  <gap reason="illegible"><?human @extent=4-6 @unit=characters ?></gap>
caes0362.xml  <gap reason="illegible"><?human @extent=2-4 @unit=characters ?></gap>
caes0363.xml  <gap reason="illegible"><?human @extent=2-3 @unit=characters ?></gap>
caes0378.xml  <gap reason="illegible"><?human @extent=1-3 @unit=characters ?></gap>
edda0001.xml  <gap reason="illegible"><?human @extent=2-3 @unit=characters ?></gap>
hamm0001.xml  <gap reason="illegible"><?human @extent=8-9 characters ?></gap>
hamm0001.xml  <gap reason="illegible"><?human @extent=8-10 characters ?></gap>
mums0003.xml  <gap reason="illegible"><?human @extent=4-6 @unit=characters ?></gap>
mums0003.xml  <gap reason="illegible"><?human @extent=4-6 @unit=characters ?></gap>
mums0003.xml  <gap reason="illegible"><?human @extent=4-6 @unit=characters ?></gap>
mums0003.xml  <gap reason="illegible"><?human @extent=4-6 @unit=characters ?></gap>
sepp0014.xml  <gap reason="illegible"><?human @extent=3-4 @unit=characters ?></gap>
sepp0017.xml  <gap reason="illegible"><?human @extent=18-20 @unit=characters ?></gap>
zoor0064.xml  <gap reason="illegible"><?human @extent=8-9 @unit=characters ?></gap>
zoor0203.xml  <gap reason="illegible"><?human @unit=characters ?></gap>
zoor0255.xml  <gap reason="illegible"><?human @extent=8-9 @unit=characters ?></gap>
zoor0255.xml  <gap reason="illegible"><?human @extent=11-12 @unit=characters ?></gap>
zoor0282.xml  <gap reason="illegible"><?human @extent=ca. 7 @unit=characters ?></gap>
zoor0283.xml  <gap reason="illegible"><?human @extent=ca. 5 @unit=characters ?></gap>
zoor0283.xml  <gap reason="illegible"><?human @extent=ca. 5 @unit=characters ?></gap>
zoor0284.xml  <gap reason="illegible"><?human @extent=ca. 6 @unit=characters ?></gap>
zoor0284.xml  <gap reason="illegible"><?human @extent=ca. 6 @unit=characters ?></gap>
zoor0291.xml  <gap reason="illegible"><?human @extent=3-4 @unit=characters ?></gap>
zoor0291.xml  <gap reason="illegible"><?human @extent=ca. 1 @unit=character ?></gap>
zoor0295.xml  <gap reason="illegible"><?human @extent=ca. 7 @unit=characters ?></gap>
zoor0311.xml  <gap reason="illegible"><?human @extent=2-3 @unit=characters ?></gap>
zoor0311.xml  <gap reason="illegible"><?human @extent=1-2 @unit=characters ?></gap>
zoor0312.xml  <gap reason="illegible"><?human @extent=ca. 17-18 @unit=characters ?></gap>
zoor0312.xml  <gap reason="illegible"><?human @extent=ca. 12-13 @unit=characters ?></gap>
zoor0332.xml  <gap reason="illegible"><?human @extent=8-9 @unit=characters ?></gap>
zoor0342.xml  <gap reason="illegible"><?human @extent=ca. 7 @unit=characters ?></gap>
zoor0354.xml  <gap reason="illegible"><?human @extent=ca. 11 @unit=characters ?></gap>
zoor0359.xml  <gap reason="illegible"><?human @extent=7-8 @unit=characters ?></gap>
```

**Comment by GaiaLembi** [comment]
If you can fix this globally, please do so.
I would agree to use precision="low" only in the second type of gap, as it seems superfluous in the first one.

**Comment by emylonas** [comment]
zoor0203 didn't have an extent. I added unknown, but it may just be missing because it's a mistake.

**Comment by emylonas** [comment]
zoor0312.xml  <gap reason="illegible"><?human @extent=ca. 17-18 @unit=characters ?></gap>
which seems best: 
quantity="17" precision="low" or
atlLeast="17 atMost="18"

**Comment by emylonas** [comment]
closed with commit 3dbf4ff
